cache/models/samples.js

```diff
@@ -647,6 +649,7 @@ module.exports = {
       }
 
       cleanQueryBodyObj(reqBody);
+      aspectObj = aspObj;
```

Comment:
Below code instead?
`         aspectObj = sampleStore.arrayStringsToJson(
            aspObj, constants.fieldsToStringify.aspect		            aspObj, constants.fieldsToStringify.aspect
         );		         );`
Not sure why it was put inside the value check.